app/src/test/java/org/fossasia/openevent/app/presenter/AttendeePresenterTest.java

```diff
@@ -227,6 +280,22 @@ public void shouldNotShowEmptyViewOnSwipeRefreshSuccess() {
     }
 
     @Test
+    public void shouldShowScanButtonOnSwipeRefreshSuccess() {
+        when(eventRepository.getAttendees(id, true))
+            .thenReturn(Observable.fromIterable(attendees));
+
+        InOrder inOrder = Mockito.inOrder(eventRepository, attendeesView);
+
+        attendeesPresenter.loadAttendees(true);
+
+        inOrder.verify(attendeesView).showScanButton(false);
+        inOrder.verify(eventRepository).getAttendees(id, true);
+        inOrder.verify(attendeesView).showAttendees(attendees);
+        inOrder.verify(attendeesView).onRefreshComplete();
+        inOrder.verify(attendeesView).showScanButton(true);
```

Comment:
This was a mistake I made in past. Please only verify that scan button is shown. Tests should not overlap. Do it with the other tests you added as well. Nice job!